README.md

```diff
@@ -167,6 +167,28 @@ await pc.createIndex({
 });
 ```
 
+#### Create a sparse serverless index
+
+You can also use `vectorType` to create `sparse` serverless indexes. These indexes enable direct indexing and retrieval of sparse vectors, supporting traditional methods like BM25 and learned sparse models such as [pinecone-sparse-english-v0](https://docs.pinecone.io/models/pinecone-sparse-english-v0). A `sparse` index must have a distance metric of `dotproduct` and does not require a specified dimension:
```

Comment:
not a huge deal, but you don't have to pass in a metric. it'll default to dotproduct.